github/scim.go

```diff
@@ -62,17 +82,22 @@ type ListSCIMProvisionedIdentitiesOptions struct {
 // ListSCIMProvisionedIdentities lists SCIM provisioned identities.
 //
 // GitHub API docs: https://docs.github.com/en/rest/scim#list-scim-provisioned-identities
-func (s *SCIMService) ListSCIMProvisionedIdentities(ctx context.Context, org string, opts *ListSCIMProvisionedIdentitiesOptions) (*Response, error) {
+func (s *SCIMService) ListSCIMProvisionedIdentities(ctx context.Context, org string, opts *ListSCIMProvisionedIdentitiesOptions) (*SCIMProvisionedIdentities, *Response, error) {
 	u := fmt.Sprintf("scim/v2/organizations/%v/Users", org)
 	u, err := addOptions(u, opts)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 	req, err := s.client.NewRequest("GET", u, nil)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
-	return s.client.Do(ctx, req, nil)
+	identities := new(SCIMProvisionedIdentities)
+	resp, err := s.client.Do(ctx, req, identities)
+	if err != nil {
+		return nil, resp, err
+	}
+	return identities, resp, nil
```

Comment:
Let's please clean this up a bit while we're here and add a blank line before lines 100, 95, and 91.

Reply:
Sure, I can do that - I actually removed the blank lines on purpose to match the formatting in the other functions.

Reply:
OK, I've fixed those, let me know if you want me to add similar blank lines in the rest of the funcs in this file, so they all match.

Reply:
> OK, I've fixed those, let me know if you want me to add similar blank lines in the rest of the funcs in this file, so they all match.

You can if you would like to... Thank you!

Reply:
All done, much better now ☺️  
You can probably add a linter rule in `.golangci.yml` to enforce that but there might be a bunch of files to fix 😅